### PR TITLE
Fixed checkbox label text overflow

### PR DESCRIPTION
**Pull request by simsine**
remind me to never do frontend again

image

### CODE REVIEW

**Comment by HenrikEngen** [comment]
Screenshot_20240408_222248_Chrome

The issue is fixed, good job! But I notice another visual bug, pirk pirk..
The space between the check boxes are not the same

**Review by HenrikEngen — CHANGES_REQUESTED** [review]
image

The issue is fixed, good job! But I notice another visual bug, pirk pirk..
The space between the check boxes are not the same

**Comment by simsine** [comment]
> image
> 
> The issue is fixed, good job! But I notice another visual bug, pirk pirk.. The space between the check boxes are not the same

image
sigh

**Review by petterroea — CHANGES_REQUESTED** [review]

File: staticContent/public/static/global.css [hunk 1]
```diff
@@ -29,6 +29,16 @@ label {
 	display: block;
 }
 
+/* This overwrites the mui class to fix checkbox label text from overflowing */
+.label-text {
```
[the remark this answers is not in the document]
Comment (simsine) [reply staticContent/public/static/global.css:33]: Not really, the label-text and mark classes are div wrappers for the component slot and checkbox respectively, which per [the source code](https://github.com/vikignt/svelte-mui/blob/4b95448fdf5a77629fe1e9d314499d97e4d77aab/src/Checkbox.svelte#L77) has no interface to be changed.
Is there something i am not seeing here?

**Review by HenrikEngen — APPROVED** [review]

File: staticContent/public/static/global.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (simsine) [reply staticContent/public/static/global.css:33]: I also tried increasing the css specificity in order to overwrite the component css class attributes but this does not work either.
!important is the only thing that works it seems

**Comment by petterroea** [comment]
I am also happy, seems like @simsine put in due dilligence. let's get this merged!

**Pull request merged**